src/twisted/internet/defer.py

```diff
@@ -856,6 +857,53 @@ def uncancel(result):
         return self
 
 
+    @classmethod
+    def fromCoroutine(cls, coro):
+        """
+        Schedule the execution of a coroutine that awaits/yields from
```

Comment:
```suggestion
        Schedule the execution of a coroutine that awaits
```

`yield from` coros are deprecated

Reply:
done